essay.tex

```diff
@@ -0,0 +1,43 @@
+
+\section*{Management Styles}
+
+Luckily there already exists a well proven model of managing styles which we can adopt to study the repercussions of each in the context of Agile game development.  Whilst the names and number of terms changes, the extreme values of the scale are usually agreed upon.  From Autocratic as the most dominant, to Laissez-faire which translates to soft touch.  Management styles are distinct from each other in the relative importance of desired goals \cite{fairweather1994goals}.  Leaders and managers are concerned with two aspects of their work.  The first of these aspects we term task accomplishment, organizing activities to perform assigned tasks.  The second aspect we term maintenance of interpersonal relationships, tending to the morale and welfare of the people in the setting \cite {eagly1990gender}.  Each manager's own style is weighted on a gradient towards one of these and it is widely accepted that the optimum management team encapsulates the entire range.  It is important here to differentiate between a management team usually operating over a long period and a single scrum master over a much shorter sprint. 
+
+\section*{An Agile Team}
+
+We should also consider the team members when choosing a way to manage them.  Studies confirm software developers typically fall into specific personality types, the most prominent being ISTJ using the Myers-Briggs Type Indicator (MBTI), introverted, highly rational and analytical `‘thinkers'' rather than `‘feelers'' \cite{wiesche2014relationship}.  Game developers are generally more abstractly artistic and thereby tend to a more extrovert nature.  An extroverted team, can be difficult to manage correctly, however the highly trained, intelligent, and capable nature of software developers counter many of these difficulties.  In a highly trained team for instance, little effort needs to be spent on training; a staple of the more dominant styles.  In fact, with often specialised roles a team may not require any personal management.  The Agile manifesto \cite {fowler2001agile} reflects this in its self-organising guidelines.  Perhaps this is some of the essence of the Scrum methodology?  Not the addition, but the exception of certain techniques and practices, given the software developing prefix.  The role of the product owner defining the rules and parameters of the project, siphoned off in portions through a scrum master to a capable team with the freedom to excel.  Perhaps then it is not the team members that need managing, but the process itself.
+
```

Comment:
Is there any evidence to support your claim that `Game developers are generally more abstractly artistic and thereby tend to a more extrovert nature.`?